assets/scripts/aircraft.js

```diff
@@ -334,10 +334,16 @@ var Model=Fiber.extend(function() {
 zlsa.atc.AircraftFlightManagementSystem = Fiber.extend(function() {
   return {
     init: function(options) {
-      this.aircraft = options.aircraft;
```

Comment:
@tedrek This is an interesting one.

I removed the aircraft object from being a property of the fms object due to the infinite inheritance (eg. aircraft has an an fms, which has an aircraft, which has an fms, etc). Is there some way to stop this? Or maybe we need a `get_aircraft()` function like the `get_airport()`? When I move the SID-flying logic from aircraft to fms, I will need to be able to reference the aircraft, and it seems like `fms.aircraft` would not be the way to go because of the issues above. Do you follow/agree/disagree/have feedback?